### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by d1eg0** [yongkangchen/remote-sync#84]
Eleven deprecation warnings are listed (such as the one below this text) and remote-sync (2.3.1) is not working with Atom 0.174.0:

atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (d:\Users\dgarciava\AppData\Local\atom\app-0.174.0\resources\app\src\atom.js:55:11)
Object.activate (d:\Users\dgarciava\.atom\packages\remote-sync\lib\RemoteSync.coffee:62:9)
```

**Comment by 7talents** [comment]
Same here - deprecation warning still there in Atom 0.177.0

**Comment by Gawdl3y** [comment]
Are these all fixed in version 2.4.0?

**Comment by 7talents** [comment]
Yes, just downloaded 2.4.0, restarted Atom and the deprecation warnings disappeared. Thanks!
